.gitignore

```diff
@@ -1,6 +1,8 @@
 build/*
 lib/gmock-1.7.0/*
 lib/pbjson/*
+lib/curlcpp/*
+lib/rapidjson-1.0.2/*
```

Comment:
Libraries of mine libaries are not my libraries.

We have dependency on rapidjson. Pbjson has dependency on rapidjson today - but it may not have it tomorrow.